src/response-base.js

```diff
@@ -70,10 +73,9 @@ ResponseBase.prototype._setHeaderProperties = function(header) {
   this.type = utils.type(ct);
 
   // params
-  const params = utils.params(ct);
-  for (const key in params) {
-    if (Object.prototype.hasOwnProperty.call(params, key))
-      this[key] = params[key];
+  this.typeParams = utils.params(ct);
```

Comment:
What do you think of this? Is this really needed? I did it because I thought some users might depend on having access to the data like before, but actually it could still be accessed using `res.headers` and is probably not used a lot (besides `charset` of course).